app/src/androidTest/java/com/example/lhi06/residue/ExampleInstrumentedTest.java

```diff
@@ -0,0 +1,26 @@
+package com.example.lhi06.residue;
```

Comment:
Can you delete this class? This is just a sample test which adds nothing to the project